### PR TITLE
Add debug sampling flag option for Zipkin traces

### PR DESCRIPTION
**Pull request by wiwa**
For certain debug use cases, we would like the Zipkin traces submitted to override sampling settings on the server by setting the `debug` flag on a trace.
This change will allow users to set a `zipkin.trace.debug` property for this purpose.

### CODE REVIEW

File: backend/src/main/scala/bloop/tracing/BraveTracer.scala [hunk 1]
```diff
@@ -4,10 +4,9 @@ import brave.{Span, Tracer}
 import brave.propagation.TraceContext
 import monix.eval.Task
 import monix.execution.misc.NonFatal
-
-import scala.util.Failure
-import scala.util.Success
+import scala.util.{Failure, Properties, Success}
```
Comment (olafurpg) [inline backend/src/main/scala/bloop/tracing/BraveTracer.scala]: nit: please configure IntelliJ to not group imports by prefix

Comment (wiwa) [reply backend/src/main/scala/bloop/tracing/BraveTracer.scala]: done

**Comment by wiwa** [comment]
Without the `debug` flag, traces submitted to the Zipkin server may be lost due to how the server samples traces. The `debug` flag will let the submission override the server's attempt to neglect it.

**Comment by olafurpg** [comment]
Ignoring unrelated CI failure
```
[error] sbt.librarymanagement.ResolveException: Error downloading com.lihaoyi:upickle-implicits_2.12:0.7.1
```

**Pull request merged**